examples/ecs/simple/.pipe.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: pipecd.dev/v1beta1
+kind: ECSApp
+spec:
+  input:
+    serviceDefinitionFile: servicedef.yaml
+    taskDefinitionFile: taskdef.yaml
+    targetGroups:
+      primary:
+        targetGroupArn: arn:aws:elasticloadbalancing:ap-northeast-1:XXXX:targetgroup/ecs-lb/YYYY
+        containerName: web
+        containerPort: 80
```

Comment:
Do we still need this targetGroups in case of using QuickSync?

Reply:
Yes, it is 🙏 When creating a task set, we have to point to the creating task set by a target group to enable those tasks to serve traffic. Without this target group setting, there is no way to access to the created task set 👀